Sources/SwiftGenKit/Parsers/Filter.swift

```diff
@@ -33,6 +33,17 @@ public struct Filter {
   }
 }
 
+extension Parser {
+  /// Create the regex for a filter for the given list of file extensions. The extensions will be matched at the end
+  /// of a filename, and will be matched case-insensitively.
+  ///
+  /// - Parameter extensions: The list of file extensions. Note that these can contain regex syntax if needed.
+  /// - Returns: The regex as a `String`.
+  static func filterRegex(for extensions: [String]) -> String {
```

Comment:
naming nitpick: wondering if the API shouldn't be called `filterRegEx(extensions:)` or `filterRegEx(forExtensions:)`.

imho seeing just `filterRegex(for: x)` at call side is not as readable and clear what's expected as parameter here